Decompiler/PscCoder.cpp

```diff
@@ -348,7 +348,12 @@ void Decompiler::PscCoder::writeProperty(int i, const Pex::Property& prop, const
 
         auto initialValue = var->getDefaultValue();
         if (initialValue.getType() != Pex::ValueType::None)
+        {
             stream << " = " << initialValue.toString();
```

Comment:
It would probably be better to fix `.toString()` to return with the right trailing decimals. I believe it should just be removing these lines, which appear to exist to remove the trailing `.0`:

https://github.com/Orvid/Champollion/blob/master/Pex/Value.cpp#L295-L304

Reply:
derp, of course. thank you.